### PR TITLE
Implement configurable sentry

### PR DESCRIPTION
**Pull request by Voltmod**
Now Sentry is used via the official SDK and can be fully configurable.

### CODE REVIEW

**Review by TrofimovAnton85 — APPROVED** [review]

**Review by aednikanov — APPROVED** [review]

File: src/plugins/sentry/sentry.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    sampleRate: isProd ? 0.25 : 0,
+    tracesSampleRate: 0,
+    debug: !isProd,
+  });
+
+  Sentry.replayIntegration({
+    maskAllText: false,
+  });
+
+  console.log("Sentry client ready");
```
Comment (AndyMBridges) [inline src/plugins/sentry/sentry.ts:27]: @Voltmod could you remove the `console.log` please

**Review by TrofimovAnton85 — DISMISSED** [review]

**Review by aednikanov — DISMISSED** [review]

File: src/theme/Root.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { useState, createContext, ReactChild } from "react";
 import { MetaMaskSDK } from "@metamask/sdk";
 import BrowserOnly from "@docusaurus/BrowserOnly";
+// import Sentry from "sentry-plugin";
```
Comment (cedricmagne) [inline src/theme/Root.tsx:4]: @Voltmod Could you remove comment

File: src/theme/Root.tsx [hunk 3]
```diff
@@ -29,6 +30,7 @@ export default function Root({ children }: { children: ReactChild}) {
         const provider = sdk?.getProvider();
         setMetaMaskProvider(provider);
       }
+      // Sentry.setUser({ email: "" })
```
Comment (cedricmagne) [inline src/theme/Root.tsx:33]: ditto

**Review by cedricmagne — COMMENTED** [review]
Screenshot 2024-09-17 at 10 21 06

**Review by aednikanov — APPROVED** [review]

**Review by TrofimovAnton85 — APPROVED** [review]

**Review by cedricmagne — APPROVED** [review]

**Pull request merged**